assets/tests/components/mapPage.test.tsx

```diff
@@ -696,7 +697,43 @@ describe("<MapPage />", () => {
     expect(mapSearchPanel).toHaveClass("c-map-page__input-and-results--visible")
   })
 
+  test("When a vehicle is selected from the list of search results on mobile, then search panel should close", async () => {
+    mockScreenSize("mobile")
+    jest.spyOn(global, "scrollTo").mockImplementationOnce(jest.fn())
+
+    const runId = runIdFactory.build()
+    const vehicle = vehicleFactory.associations({ runId }).build()
+
+    mockVehicleSearchResultsCategory([vehicle])
+    mockUseVehicleForId([vehicle])
+    mockUseVehiclesForRouteMap({ [vehicle.routeId!]: [vehicle] })
+
+    render(
+      <RealDispatchWrapper
+        initialState={stateFactory.build({
+          searchPageState: activeSearchPageStateFactory.build({
+            query: { text: vehicle.runId! },
+          }),
+        })}
+      >
+        <MapPage />
+      </RealDispatchWrapper>
+    )
+
+    const mapSearchPanel = getMapSearchPanel()
+    expect(mapSearchPanel).toHaveClass("c-map-page__input-and-results--visible")
```

Comment:
**suggestion (non-blocking):** I'd argue that we could remove this assertion since it's not actually related to the property being tested (that the drawer gets closed after selecting a result) - it's more testing the setup than the actual result.

Reply:
Since we cannot control the visible state of the panel via props or state, I'd argue that this is necessary to test that it's closing, because if this assumes that it starts at `--visible`, but it actually starts with `--hidden` then this test would silently succeed when the functionality may be failing.

This test file is fairly large though, so if another test is asserting that this same setup and state opens as `--visible`, I'd agree that it could be removed.

Reply:
The reason I added it was what Kayla said - basically to highlight the contrast. I don't feel that strongly about it, but I also don't think that it's asserted elsewhere.

That highlighted contrast could be in another test right next to this one though. Let's see how that looks.

Reply:
Hmmm - okay actually... two thoughts:

1. I copied the structure of this test from the one above it, and I worry about confusion if we have a bunch of tests right near each other that are structured differently. 
2. Splitting this test up will increase the length of the test file. Not necessarily the worst thing in the world, but I do want to be considerate about that kind of thing.

Reply:
Yeah, those are both good points. I'm fine with keeping it as-is, but I wanted to throw it out there as an idea. Maybe something to consider as we write tests in the future.